lib/tdf3/src/client/index.ts

```diff
@@ -271,9 +271,10 @@ export class Client {
       this.kasEndpoint = clientConfig.keyRewrapEndpoint.replace(/\/rewrap$/, '');
     }
 
+    const kasOrigin = new URL(this.kasEndpoint).origin;
     if (clientConfig.allowedKases) {
-      this.allowedKases = [...clientConfig.allowedKases];
-      if (!validateSecureUrl(this.kasEndpoint) && !this.allowedKases.includes(this.kasEndpoint)) {
+      this.allowedKases = clientConfig.allowedKases.map((a) => new URL(a).origin);
```

Comment:
Do we care at this time about the k8s issue where services might create TDFs pointing to the internal cluster URL (on http or with an internal port) that a browser might be unable to resolve or that might be different from outside the cluster? Is it crucial to have this validation in the frontend?